utkarsh component/toast/toast.css

```diff
@@ -0,0 +1,625 @@
+}
+
+
+hr{
+    border-top: 1px solid var(--primary-color); 
+  width: 100rem;
+}
+
+.highlight{
+    color: var(--secondary-color);
+}
+
+iframe{
+    padding-top: 1rem;
+}
+
+
+
+.avatar-image {
+    width: 3rem;
+    height: 3rem;
+    border-radius: 50%;
+    position: relative;
+}
+
+.medium{
+    width: 4.5rem;
+    height: 4.5rem;
+    margin-left: 2rem; 
+}
+
+.large{
+    width: 5.5rem;
+    height: 5.5rem;
+    margin-left: 2rem; 
+}
+
+
+
+footer{
+    grid-area: footer;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 0.5rem;
+      
+
+}
+
+footer ul{
+ padding-inline-start: 0.5rem;
+    display: flex;
+  gap: 20px;
+
+
+}
+
+
+
+
+
+
+
+     div {
+        width: 70px;
+    } 
+    
+    
+     a:hover
+    {
+        color: #030;
+        border: 1px solid #9999ff;
+    }
+    
+    a:active
+    {
+        color: #aca;
+        border-left: 2px solid #030;
+        border-top: 2px solid #030;
+        border-right-width: 0px;
+        border-bottom-width: 0px;
+    } 
+
+    .navbar-responsive {
+        padding: 1rem;
+        background-color: var(--body-background-color);
+        color: var(--text-color-black);
+        box-shadow: var(--box-shadow-down);
+        padding: 0.3rem 1.2rem;
+        margin-bottom: 1rem;
+      }
+      
+      ul {
+        list-style-type: none;
+      }
+      
+      a {
+        color: var(--text-color-black);
+        text-decoration: none;
+      }
+      
+      .menu li {
+        font-size: 1.2rem;
+        font-weight: 500;
+        padding: 0.9rem 0.3rem;
+      }
+      
+      .menu li a {
+        display: block;
+      }
+      
+      .logo a {
+          font-size: 1.4rem;
+          font-weight: 500;
+          color: var(--primary-color);
+      }
+      
+
+      .menu {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-between;
+        align-items: center;
+      }
+      
+      .toggle {
+        order: 1;
+      }
+      
+      .item-nav.button {
+        order: 2;
+      }
+      
+      .item-nav {
+        width: 100%;
+        text-align: center;
+        order: 3;
+        display: none;
+      }
+      
+      
+      .item-nav.active {
+        display: block;
+      }
+      
+      .toggle {
+        cursor: pointer;
+      }
+      
+      .bars {
+        background: var(--hover-color);
+        display: inline-block;
+        height: 2px;
+        position: relative;
+        width: 1.1rem;
+      }
+      
+      .bars::before,
+      .bars::after {
+        background: var(--hover-color);
+        content: "";
+        display: inline-block;
+        height: 2px;
+        position: absolute;
+        width: 1.1rem;
+      }
+      
+      .bars::before {
+        top: 0.3rem;
+      }
+      
+      .bars::after {
+        top: -0.3rem;
+      }
+      
+      /* Tablet Menu */
+      @media all and (min-width: 468px) {
+        .menu {

```

Comment:
kindly do not  use media queries anywhere in the middle of your CSS and always ensure that all of them are arranged in the  end of your CSS file